setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import find_packages, setup
+
+install_requires = [
+    "pandas>=1.2.5,<2.0",
+    "boto3<=1.26.55",
```

Comment:
add a lower bound, I assume you don't want someone having boto3==0.1.
Skip the rest